pkg/controller/migplan/storage.go

```diff
@@ -26,9 +26,12 @@ func (r ReconcileMigPlan) ensureStorage(plan *migapi.MigPlan) error {
 	if err != nil {
 		return liberr.Wrap(err)
 	}
-	if storage == nil || !storage.Status.IsReady() {
+	if storage == nil {
 		return nil
 	}
+	if !storage.Status.IsReady() {
+		return liberr.Wrap(r.deleteImageRegistryResources(plan))
```

Comment:
This is purely a convention thing, but I think pretty much everywhere in the controller we do 

```
err := someFunc()
return liberr.Wrap(err)
```

Obviously the two are equivalent, I think code readability improves with consistency.